scipy/optimize/_minimize.py

```diff
@@ -56,9 +56,14 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
         where ``x`` is an 1-D array with shape (n,) and ``args``
         is a tuple of the fixed parameters needed to completely
         specify the function.
-    x0 : ndarray, shape (n,)
```

Comment:
`minimize` is used for many different methods therefore any specific documentation for Nelder Mead is added there, and not here. Could you remove this change?